### ISSUE TITLE
"Error decrypting save file"

### ISSUE THREAD

**Issue by smdlion** [connordennison/phasmoeditor#16]
I have bad news, either your project can't decrypt save file anymore or after several edits of file I forgot something..

https://github.com/connordennison/phasmoeditor/assets/74847386/ce26ae05-b4c9-47c8-a1fa-ddcf43a7d8f8

![image](https://github.com/connordennison/phasmoeditor/assets/74847386/3782b9e8-50c9-419f-9f25-c7725408c13e) 
the path I used

**Comment by cnnrde** [comment]
Looks like your file could be corrupt - just tested on latest ver and it seems to work fine:
Try deleting SaveFile.txt, Player.log, Player-prev.log - and then launch Phasmophobia. It'll recreate the SaveFile and you should be able to edit again

**Comment by smdlion** [comment]
Sorry but I need to put here a link so you can look at the problem better. I just don't know what to do and how I "corrupted" the file..
https://www.youtube.com/watch?v=yj9Ir_iCJ2w

**Comment by cnnrde** [comment]
Have you disabled steam cloud? Sometimes (from my testing) something just tends to go wrong and the save files become broken to some extent. I can upload a clean save file if need be that you can use as a base

**Comment by smdlion** [comment]
Yeah, the first three times I did it with steam cloud, then turned it off and did several times without it and recorded one for you, so it didn't help aswell as you can see in the video

**Comment by smdlion** [comment]
Yep, my file is corrupted and I don't know how to recover it. I guess I'll need to look for some saves from other people. Do you maybe know how can I recover it from corruptness?

**Comment by smdlion** [comment]
Well.. Somehow I fixed my save and I can use editor now

**Comment by ruslat88** [comment]
Hello, I have the same error, but I can't solve it, can you help me?

**Comment by smdlion** [comment]
I don't really remember how I fixed that problem but try to play some matches in phasomphobia. That should fix it
